web-ui/src/components/snackbar/SnackBarWithContext.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
+import SnackBar from "./SnackBar";
+import { AppContext, UPDATE_TOAST } from "../../context/AppContext";
+
+const SnackBarWithContext = () => {
+  const { state, dispatch } = useContext(AppContext);
+  window.snackDispatch = dispatch;
+  const { severity, toast } = state.toast;
+
+  const closeToast = () => {
+    dispatch({
+      type: UPDATE_TOAST,
+      payload: {
+        severity: "",
+        toast: "",
+      },
+    });
+  };
+  return (
+    <SnackBar
+      handleClose={closeToast}
+      open={toast !== "" && severity !== ""}
+      toast={toast}
+    />
+  );
+};
+
+export default SnackBarWithContext;
```

Comment:
Any reason not to combine this file and the one above? I can't see a situation where you would use the inner `SnackBar` by itself, right?